lib/platform/bitbucket/index.ts

```diff
@@ -712,16 +712,50 @@ export async function updatePr({
     )
   ).body;
 
-  await bitbucketHttp.putJson(
-    `/2.0/repositories/${config.repository}/pullrequests/${prNo}`,
-    {
-      body: {
-        title,
-        description: sanitize(description),
-        reviewers: pr.reviewers,
-      },
-    }
-  );
+  try {
+    await bitbucketHttp.putJson(
+      `/2.0/repositories/${config.repository}/pullrequests/${prNo}`,
+      {
+        body: {
+          title,
+          description: sanitize(description),
+          reviewers: pr.reviewers,
+        },
+      }
+    );
+  } catch (err) /* istanbul ignore next */ {
+    logger.warn({ err }, 'PR contains inactive reviewer accounts.  Will try setting only active reviewers');
+
+    // Bitbucket returns a 400 if any of the PR reviewer accounts are now inactive (ie: disabled/suspended)
+    let activeReviewers: Array<any> = new Array();
+
+    // Validate that each previous PR reviewer account is still active
+    pr.reviewers.forEach(async (reviewer, index) => {
```

Comment:
This won't work, as the forEach won't wait for the async callbacks.

Reply:
thx - replaced with a for...of Loop